static/js/src/dynamic-forms.js

```diff
@@ -104,6 +105,7 @@ import { prepareInputFields } from "./prepare-form-inputs.js";
 
     // Open the contact us modal
     function open() {
+      modalTrigger.setAttribute("aria-expanded", "true");
```

Comment:
There is a potential that `modalTrigger` is assigned `document.body`. In this situation, we probably don't want to set the body as `aria-expanded="true"`